charts/seed-bootstrap/charts/kube-state-metrics/templates/kube-state-metrics.yaml

```diff
@@ -0,0 +1,117 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kube-state-metrics
+  namespace: {{ .Release.Namespace }}
+  labels:
+    component: kube-state-metrics
+    type: seed
+
+---
+apiVersion: {{ include "rbacversion" . }}
+kind: ClusterRoleBinding
+metadata:
+  name: gardener.cloud:monitoring:kube-state-metrics
+  labels:
+    component: kube-state-metrics
+    type: seed
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: kube-state-metrics
```

Comment:
This should be prefixed as well, I guess.

Reply:
But the `ClusterRole` [already exists](https://github.com/gardener/gardener/blob/master/charts/seed-bootstrap/templates/clusterrole-kube-state-metrics.yaml#L8) and is already used in [seed monitoring](https://github.com/gardener/gardener/blob/master/charts/seed-monitoring/charts/core/charts/kube-state-metrics-seed/templates/kube-state-metrics.yaml#L22). Should I still change it?

Reply:
I see.. now, it's okay then. Thanks.